packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -267,6 +280,36 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('debounceSaytRequests', () => {
```

Comment:
Most of our descriptions for tests against methods have been using the `()` after them. So:
```suggestion
  describe('debounceSaytRequests()', () => {
```